### ISSUE TITLE
Docker build broken and npm package mismatch potential

### ISSUE THREAD

**Issue by mtheoryx** [noopkat/electric-io#74]
**TLDR**: I already have a fix for this, but wanted to record this correctly as an issue. I'll take it asap.

## Docker node_modules are just a whole buncha messed up

This is my bad. What's happening currently is that on build-time, docker seems to be working fine. However, on run-time, where the volume mapping comes in, we end up with new-hotness from several PRs ago not being reflected. And this is not 100% reproducible, depending on whether you already had this stuff installed on your dev machine.

Fix is to correctly map app and node_modules volumes in docker-compose, in addition to copying in the package-lock.json into the container for the new build step.

## Version mismatch between VUE and vue-template-compiler

Vue was at `2.5.16` as a dependency and vue-template-compiler was added at `~2.5.16`. The latter issued a patch fix `~2.5.17` and appeared to be causing an error.

Fix is to lock down vue-template-compiler to `2.5.16` to keep in step with vue releases.
